api/serializers.py

```diff
@@ -283,8 +283,20 @@ class JSONAPIMeta:
 
 
 class ItemSerializer(CollectionModelSerializer):
-    type = ChoiceField(
-        choices=['none', 'project', 'preprint', 'registration', 'presentation', 'website', 'event', 'meeting'])
+    kind = ChoiceField(
```

Comment:
it might be better to just import this from the model

Reply:
I agree. That's part of why I moved the lists out of the class. I'll probably do that when I go through and do so deep cleanup of the backend.

Reply:
would you be down to do that as part of this PR? i'd feel much more comfortable doing our best to make sure all code being merged in is clear, maintainable, and as good as we can get it, and not waiting if it's a quick fix

Reply:
hmmmm ok

Reply:
awesome, thank you!